src/components/Contacts/ContactListTable.jsx

```diff
@@ -44,31 +50,74 @@ const ContactListTable = ({ contacts, deleteContact }) => {
   const contactsCopy = [...contacts];
   const sortedContacts = contactsCopy.sort(comparePerson);
```

Comment:
With the new data grid, is a custom sorting function still necessary? Unless I'm mistaken, it should be possible to use `initialState` in `DataGrid` for the same results.

Reply:
We could certainly drop the custom sort and leave it as an initial state.

Reply:
Got the custom sort dropped for sorting from `initialState`